test/itkCannyEdgesDistanceAdvectionFieldFeatureGeneratorTest1.cxx

```diff
@@ -76,30 +75,35 @@ int itkCannyEdgesDistanceAdvectionFieldFeatureGeneratorTest1( int argc, char * a
 
   featureGenerator->SetInput( inputObject );
 
+
+  double sigma = 1.0;
```

Comment:
I tried to check them as systematically as I was able to. In this case, it is:
https://github.com/InsightSoftwareConsortium/LesionSizingToolkit/blob/459d226d5d7f2212fdac54900ded5fb28f884736/include/itkCannyEdgesDistanceAdvectionFieldFeatureGenerator.hxx#L46

Unfortunately, default values do not exist for all member variables, and hence, once the CI is enabled, this will give raise to errors. But I find that if we do not have a way to measure them, we will just not be aware of them.